Resources/Prototypes/Entities/Objects/Specific/Janitorial/janitor.yml

```diff
@@ -203,3 +203,30 @@
           maxFillLevels: 3
           fillBaseName: cart_water_
           changeColor: false
+
+
+- type: entity
+  id: FloorDrain
+  name: drain
+  description: Drains puddles around it. Useful for dumping mop buckets or keeping certain rooms clean.
+  placement:
+    mode: SnapgridCenter
```

Comment:
this should be parented to basestructure or smth right? if not, needs to be transform anchored